temporalio/service.py

```diff
@@ -710,6 +710,31 @@ def __init__(self, client: ServiceClient) -> None:
             wsv1.UpdateWorkflowExecutionOptionsRequest,
             wsv1.UpdateWorkflowExecutionOptionsResponse,
         )
+        self.create_workflow_rule = client._new_call(
+            "create_workflow_rule",
+            wsv1.CreateWorkflowRuleRequest,
+            wsv1.CreateWorkflowRuleRequest,
+        )
+        self.delete_workflow_rule = client._new_call(
+            "delete_workflow_rule",
+            wsv1.DeleteWorkflowRuleRequest,
+            wsv1.DeleteWorkflowRuleResponse,
+        )
+        self.describe_workflow_rule = client._new_call(
+            "describe_workflow_rule",
+            wsv1.DescribeWorkflowRuleRequest,
+            wsv1.DescribeWorkflowRuleResponse,
+        )
+        self.list_workflow_rules = client._new_call(
+            "list_workflow_rules",
+            wsv1.ListWorkflowRulesRequest,
+            wsv1.ListWorkflowRulesResponse,
+        )
+        self.trigger_workflow_rule = client._new_call(
+            "trigger_workflow_rule",
+            wsv1.TriggerWorkflowRuleRequest,
+            wsv1.TriggerWorkflowRuleResponse,
+        )
```

Comment:
You may need to also add these in `client.rs` (sorry it's such a manual process, we didn't expect RPCs to get added so regularly to justify auto-generating this, though we probably should)

Reply:
added
from what I can tell, `test_all_grpc_calls_present` only compares the python client vs the GRPC service it should implement, there's not comparison with the rust bridge client
```
    assert_all_calls_present(
        client.workflow_service,
        temporalio.api.workflowservice.v1,
        temporalio.api.workflowservice.v1.WorkflowServiceStub,
   )
```

Reply:
Ah, yeah, it's just testing the pure Python gRPC client stub creation matches what we've set in Python, it does not actually make the call like some SDKs do